src/object.rs

```diff
@@ -0,0 +1,282 @@
+//! Any APRS station can manually report the position of an APRS entity (e.g.
+//! another station or a weather phenomenon). This is intended for situations
+//! where the entity is not capable of reporting its own position.
+//! Object Reports specify an Object’s position, can have an optional timestamp,
+//! and can include course/speed information or other Extended Data. Object
+//! Reports are intended primarily for plotting the positions of moving objects
+//! (e.g. spacecraft, storms, marathon runners without trackers).
+
+use std::convert::TryFrom;
+use std::io::Write;
+
+use Callsign;
+use DecodeError;
+
+use EncodeError;
+use Timestamp;
+
+use crate::{AprsCst, AprsPosition};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct AprsObject {
+    pub to: Callsign,
+    pub name: Vec<u8>,
+    pub live: bool,
+    pub timestamp: Timestamp,
+    pub position: AprsPosition,
+}
+
+impl AprsObject {
+    pub fn comment(&self) -> &[u8] {
+        &self.position.comment
+    }
```

Comment:
Sorry, one more nitpick. Thanks for adding this method! Could we add a newline after the `}`?